Makefile

```diff
@@ -116,6 +116,7 @@ manifests: controller-gen ## Generate manifests e.g. CRD, RBAC etc.
 # Run e2e tests locally using go build while also setting up a proxy in the shell to allow
 # the test to run as if it were inside the cluster. This enables mongodb connectivity while running locally.
 e2e-telepresence: cleanup-e2e install ## Run e2e tests locally using go build while also setting up a proxy e.g. make e2e-telepresence test=replica_set cleanup=true
+	export MDB_LOCAL_OPERATOR=true; \
```

Comment:
exporting this means we are expecting folks to run the test locally (go test) **and** that the operator runs locally as well. Maybe we should make a new target for this? Or maybe a comment is enough 🤔